app/Models/Relations/UserRelations.php

```diff
@@ -166,4 +166,13 @@ public function bragiLogs()
     {
         return $this->hasMany(BragiLog::class);
     }
+
+    /**
+     * List of subscription cancellations for the user
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function cancellations()
+    {
+        return $this->hasMany('App\Models\SubscriptionCancellation', 'user_id', 'id');
```

Comment:
I don't see a `where()` to limit it on cancelled subscriptions? Currently it looks for any kind of subscription, even failed or ongoing ones